doc/users-guide/base/typeAliases.rst

```diff
@@ -0,0 +1,67 @@
+:title: Users Guide: Type Aliases
+
+.. _ug-typeAliases:
+
+Type Aliases
+============
+
+In addition to :ref:`declaring variables <ug-vars>` using ``var`` and
+:ref:`constants <ug-constParam>` using ``const`` and ``param``, Chapel
+supports the declaration of *type aliases* using the ``type`` keyword.
+As the term suggests, type aliases create symbolic names for types.
```

Comment:
I worry that these counterproposals overload the 'type' keyword too much, but I think you're right that we should move this discussion to its own issue rather than belaboring it here.  That said, I can't resist saying that part of the reason I like using the ':' for such cases is that it suggests a "subtype" relationship while also providing a nice way to define default values for new types, if we wanted to support it:

    typedef age: int = 21;